src/main/java/storage/Storage.java

```diff
@@ -0,0 +1,112 @@
+package storage;
+
+import java.io.IOException;
+import java.time.format.DateTimeFormatter;
+
+import task.Deadline;
+import task.Event;
+import task.Task;
+import task.TaskList;
+import task.Todo;
+import utility.DateTimeParser;
+import utility.TextFileHandler;
+
+/**
+ * Utility class for reading and writing tasks to/from a textfile.
+ *
+ * @author Ho Khee Wei
+ */
+public abstract class Storage {
+    /** Path to the textfile. */
+    public static final String TASK_FILE_PATH = "data/tasks.txt";
+
+    /** Character indicator that separates one elements in the textfile. */
+    public static final String TASK_FILE_SEPARATOR = "-";
+
+    /** Standard format for time and date. */
+    public static final String TASK_FILE_TB_FORMAT = "dd/MM/yyyy HHmm";
```

Comment:
does it need to be public or can it be changed to private?

Reply:
For now, no.  But since its `static` `final`, other classes can safely access and I do see the possibility that other class might need it in the future. I guess I can change the access modifier to `private` for now.